### PR TITLE
feat: client sdk - support new media sets

### PR DESCRIPTION
**Pull request by krzysu**
similar as in react sdk, using config to setup defaults, but defaults are empty, so using API default values

### CODE REVIEW

File: packages/client/src/feed/Feed.ts [hunk 1]
```diff
@@ -44,6 +50,7 @@ export class Feed {
   async fetch(
     request: FeedRequest,
     observerId?: string,
+    mediaTransformParams: MediaTransformParams = defaultMediaTransformParams,
```
Comment (desfero) [inline packages/client/src/feed/Feed.ts:53]: @cesarenaldi Are we okay with just allowing you to have one transformed image?

Comment (krzysu) [reply packages/client/src/feed/Feed.ts:53]: in react-sdk PR we decided to introduce separate image sets for Publication and two sets for Profile (picture and cover) 
see: https://github.com/lens-protocol/lens-sdk/pull/430/files#diff-b89498756162aec8b024c715e6744fceac12f4529c23d825c4baafba133e2f45

I wanted to keep it simple with client sdk, but we could do sth similar here too

Comment (krzysu) [reply packages/client/src/feed/Feed.ts]: I rewrite the whole thing, could you take a look again?

File: packages/client/codegen-api.yml [hunk 2]
```diff
@@ -24,12 +24,12 @@ config:
     FollowModuleData: string
     Handle: string
     HandleClaimIdScalar: string
-    IpfsCid: string
+    ImageSizeTransform: string
```
Comment (cesarenaldi) [inline packages/client/codegen-api.yml:27]: Can we copy over from react package the more type safe version of `ImageSizeTransform`?

**Review by cesarenaldi — COMMENTED** [review]
Just one question

File: packages/client/codegen-api.yml (hunk 2 above)
Comment (krzysu) [reply packages/client/codegen-api.yml]: ok, I added it for both api-bindings and client packages 

https://github.com/lens-protocol/lens-sdk/pull/424/commits/ebfa1011e39f64501eed074f2780792ed067d782

File: packages/client/src/revenue/Revenue.ts [hunk 3]
```diff
@@ -115,6 +121,7 @@ export class Revenue {
         {
           request,
           observerId,
+          ...buildMediaTransformsFromConfig(this.config.mediaTransforms),
```
Comment (reecejohnson) [inline packages/client/src/revenue/Revenue.ts:124]: I wonder if there's a nice way to abstract this enriching of the request into a function `...buildMediaTransformsFromConfig(this.config.mediaTransforms)` and keep this logic contained into one place?

Comment (reecejohnson) [reply packages/client/src/revenue/Revenue.ts:124]: `enrichRequestWithMediaTransformations(...)` maybe?

**Review by reecejohnson — APPROVED** [review]

File: packages/client/src/revenue/Revenue.ts (hunk 3 above)
Comment (reecejohnson) [reply packages/client/src/revenue/Revenue.ts:124]: discussed with @krzysu, difficult to abstract any further due to media transformations not always being required

**Review by cesarenaldi — APPROVED** [review]
LGTM

**Pull request merged**